tests/functional/model/models_information.py

```diff
@@ -109,8 +101,7 @@ class Resnet:
     output_shape = (1, 1001)
     rest_request_format = 'column_name'
     model_path = os.path.join(config.models_path, name)
-    url = URL_PUBLIC_MODEL_FORMAT.format(repo=MODEL_REPOSITORY_SERVER, opencv=OPENCV_PUBLIC, build=BUILD_012020,
-                                         model=RESNET_50, model_version=RESNET_V1_50)
+    url = "https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1/resnet-50-tf/" + RESNET_V1_50
```

Comment:
url = f"https://storage.openvinotoolkit.org/repositories/open_model_zoo/public/2022.1/resnet-50-tf/{RESNET_V1_50}"

Reply:
/2022.1/ is correct?